lib/Service/ColumnTypes/NumberStarsBusiness.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+namespace OCA\Tables\Service\ColumnTypes;
+
+use OCA\Tables\Db\Column;
+
+class NumberStarsBusiness extends SuperBusiness implements IColumnTypeBusiness {
+
+	public function parseValue(string $value, ?Column $column = null): string {
+		return json_encode(intval($value));
```

Comment:
at least in PHP 7 times, casting (`(int)$foo`) was faster then using `intval($foo)`. We changed it in server everywhere, and for all types.